src/main/java/org/opentripplanner/middleware/triptracker/ManageLegTraversal.java

```diff
@@ -23,29 +23,23 @@ private ManageLegTraversal() {
     }
 
     /**
-     * Define the shortest distance to any position within a trip by checking the traveler's position against all
-     * positions within all legs.
-     *
-     * @param currentCoordinates Traveler's current position.
-     * @param itinerary Trip itinerary containing all trip legs.
-     * @return The shortest distance to any trip position.
+     * Define the segment within a leg that is the closest to the traveler's current position.
      */
-    public static double getDistanceFromNearestTripPosition(Coordinates currentCoordinates, Itinerary itinerary) {
+    public static ManageLegTraversal.Segment getLegSegmentNearestToCurrentPosition(Leg leg, Coordinates currentCoordinates) {
```

Comment:
Do we need to qualify `Segment`?

Reply:
Addressed in https://github.com/ibi-group/otp-middleware/pull/214/commits/d4cc0bb523319ae7ee4eaa9d70446a8aa7850ee8